packages/driver/src/cypress/command.ts

```diff
@@ -3,9 +3,11 @@ import utils from './utils'
 
 export class $Command {
   attributes!: Record<string, any>
+  state: 'queued' | 'pending' | 'passed' | 'recovered' | 'failed' | 'skipped'
```

Comment:
Transparency - this helped with debugging. All we really need to care about is `skipped` & `passed`.